snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedTerminologyComponentConstants.java

```diff
@@ -51,6 +51,8 @@ private SnomedTerminologyComponentConstants() {}
 	public static final short CONSTRAINT_NUMBER = 105;
 	public static final String TEMPLATE = "com.b2international.snowowl.terminology.snomed.template";
 	public static final short TEMPLATE_NUMBER = 106;
+	public static final String COMPARE_RESULT = "com.b2international.snowowl.terminology.snomed.compare";
+	public static final short COMPARE_RESULT_NUMBER = 106;
```

Comment:
Why do you need this? Also, this uses the same value as the `TEMPLATE_NUMBER` constant.

Reply:
I used it for creating a component URI for the editor, but I will use the unspecified id instead.